ArticlesDatabase/Sources/ArticlesDatabase/ArticlesTable.swift

```diff
@@ -402,6 +412,52 @@ final class ArticlesTable: DatabaseTable {
 		}
 	}
 
+	func fetchUnreadCountBetween(webFeedIDs: Set<String>, since: Date? = nil, before: Date? = nil, completion: @escaping SingleUnreadCountCompletionBlock) {
+		// Get unread count between dates
```

Comment:
Switches from positional parameters (borrowed implementation from `fetchUnreadCount`) to named parameters. `since` and `before` are optional and in the middle of the signature but because they are named parameters they don't pose issues with argument entry order.